lib/Common/Memory/HeapBucket.cpp

```diff
@@ -379,6 +380,10 @@ HeapBucketT<TBlockType>::TryAllocFromNewHeapBlock(Recycler * recycler, TBlockAll
     // We just created a block we can allocate on
     char * memBlock = allocator->SlowAlloc<false /* disallow fault injection */>(recycler, sizeCat, attributes);
     Assert(memBlock != nullptr || IS_FAULTINJECT_NO_THROW_ON);
+    if (recycler->IsConcurrentSweepExecutingState())
```

Comment:
That will be counting all page count allocated. if there's intensive allocation before starting the background sweeping, and then right after started background sweeping it can reach the limit and result in wait. it will increase the chance to hit the waiting code path but how much chance it increases is uncertain, so result in high variation in perf. and need to set a bigger cap but not easy to find a good number for the cap